lib/app/utils/valueset.rb

```diff
@@ -32,13 +32,15 @@ class ValueSet
         'http://loinc.org' => 'LNC',
         'http://snomed.info/sct' => 'SNOMEDCT_US',
         'http://www.icd10data.com/icd10pcs' => 'ICD10PCS',
```

Comment:
ICD10PCS url is changed to 'http://www.cms.gov/Medicare/Coding/ICD10' in USCore 3.1.1